mw-webapp/src/component/form/formField/fieldContainer/FieldContainer.tsx

```diff
@@ -0,0 +1,38 @@
+import {FormLabel} from "src/component/form/formField/fieldContainer/formLabel/FormLabel";
+import {FormMessage} from "src/component/form/formField/fieldContainer/formMessage/FormMessage";
+import styles from "src/component/form/formField/fieldContainer/FieldContainer.module.scss";
+
+interface FieldContainerProps{
+  /**
+  * Content for label
+  */
+  labelContent:string;
+  /**
+  * Message if you have not entered data in the field
+  */
+  messageValueMissing:string;
+  /**
+  * Message if input data is not valid
+  */
+  messageTypeMismatch:string;
+}
+
+/**
+ * Component contains label,messages for input data
+ */
+export const FieldContainer:React.FC<FieldContainerProps> = (props:FieldContainerProps) => {
+
+  return(
+    <div className={styles.container}>
+      <FormLabel>
```

Comment:
it seems to me that this component is completely superfluous here, and you can use {Label as LabelForm} directly here